Gems/ROS2/Code/Source/VehicleDynamics/VehicleModelComponent.cpp

```diff
@@ -51,7 +55,12 @@ namespace ROS2::VehicleDynamics
                         AZ::Edit::UIHandlers::Default,
                         &VehicleModelComponent::m_vehicleConfiguration,
                         "Vehicle settings",
-                        "Vehicle settings including axles and common wheel parameters");
+                        "Vehicle settings including axles and common wheel parameters")
+                    ->DataElement(
+                        AZ::Edit::UIHandlers::Default,
+                        &VehicleModelComponent::m_manuallyControled,
+                        "ManualControl",
```

Comment:
This field is user-readable, use spaces. Also, this could be named as to suggest that the manual control is possible once this is on, but it does not prevent autonomy (i.e. the vehicle can still be controlled by the nav stack when this is on). Description is alright at the moment.